### ISSUE TITLE
Public DNSChain servers do not work ?

### ISSUE THREAD

**Issue by Doimatcuabautroi** [okTurtles/dnschain#184]
192.184.93.146 and 54.85.5.167 isn't working

**Comment by taoeffect** [comment]
Yep we had to move/shutdown the `192.184.93.146` server a while ago because of sysadmin upgrades. I don't know what happened to `54.85.5.167`. Best thing is to run your own! Also, see #180 for project status updates.

Since I am not actively working on DNSChain at the moment I will leave this issue open so that others can see it and benefit from the answer.
